openerp/addons/base/module/module.py

```diff
@@ -293,6 +293,7 @@ def _get_icon_image(self, cr, uid, ids, field_name=None, arg=None, context=None)
         'application': fields.boolean('Application', readonly=True),
         'icon': fields.char('Icon URL'),
         'icon_image': fields.function(_get_icon_image, string='Icon', type="binary"),
+        'installation_date': fields.datetime(string='Installation Date'),
```

Comment:
installed_date (not a installation_date). it should be date field, not a datetime.
and declare after installed_version field